### DISCUSSION TITLE
what are the limits to this API?

### DISCUSSION THREAD

**Discussion by koolinoor** [ranaroussi/yfinance#1513] [Q&A]
for example. how many .info requests can I send in one minute, or hour?
the same goes for price data, e.g. 1min interval

**Comment by evgesha14** [comment]
same question here

**Comment by eabase** [comment]
Did you look? 

![image](https://user-images.githubusercontent.com/52289379/146657564-be7a2bd9-c674-422a-986e-904902aae7f8.png)

Related issue #841

**Reply by HadarGabbay-PWY** [reply]
hey what is that?

**Comment by yonlugoh** [comment]
@eabase @ranaroussi And which plan would this API allow for? From my investigation, its definitely not the basic one

**Comment by ahnazary** [comment]
Yahoo has the following limitations with regards to API calls:

<img src="https://github.com/ranaroussi/yfinance/assets/49793253/a1be377e-13f4-49ac-91bf-90777d41bc41" width="660" height="700">. 
<br /><br />
Source: [Yahoo's docs](https://developer.yahooinc.com/dsp/api/docs/traffic/info/rate-limits.html)

**Reply by WiseMarius** [reply]
That's for Yahoo DSP Traffic API not for Yahoo Finance.

**Reply by elsolo5000-2** [reply]
if anyone have tried higher / lower limits i m interested to know?

i am actually trying those dsp get limits, 
if they limit me i will try the lower put and post limits and give you the result. 

actually the minute and hourly get limits do not pose problems.

**Comment by elsolo5000-2** [comment]
my script was interupted by something but i wa able to make 4400+ request without problem a day with 60/minute and 360/hour. so my guess is that you can push more.

i do not encourage you to do so because i think the more we respect the more probability they will not treat  this as a problem, the more we gonna be able to enjoy those api without problems.

**Reply by correspond9** [reply]
Were they websocket or API calls?

**Reply by elsolo5000-2** [reply]
api calls

**Comment by xtwi1** [comment]
switched my scripts to financialdata.net recently, doing 30 calls/sec and no issues so far
